internal/graph/resolvers/query.go

```diff
@@ -51,7 +52,7 @@ type query struct {
 }
 
 // Recursively collect `CrossplaneResourceTreeNode`s from the given KubernetesResource
-func (r *query) getAllDescendant(ctx context.Context, res model.KubernetesResource, parentID *model.ReferenceID) ([]model.CrossplaneResourceTreeNode, error) { //nolint:gocyclo
+func (r *query) getAllDescendant(ctx context.Context, res model.KubernetesResource, parentID *model.ReferenceID) []model.CrossplaneResourceTreeNode { //nolint:gocyclo
```

Comment:
It looks like we're opting to not return the errors due to them being included in the error responses. Do we want to log the error at all?

Reply:
I don't think adding more logging here is going to be helpful but I'm happy to do that in a separate PR.